### PR TITLE
Milestone 1: React-Task-Manager - feature/task-manager-app

### PR DESCRIPTION
**Pull request by ITurres**
# 🚩: Milestone1 : Raact-Task-Manager | development - >>branch<< -

---

## \*📂 root/

- ### \*📂 src/components/

  - \*\*📄 Header.jsx

    > This component will render the header of the application.

  - \*\*📄 InputTaskManager.jsx

    > For now this component will render the `div` that will contain the `input` and `button` elements in the future.

  - \*\*📄 TaskManagerApp.jsx

    > This component will render the `Header` and `TaskManagerLogic` components.

  - \*\*📄 TaskManagerItem.jsx

    > This component will render the `li` element with all the data passed as props.
    > Here I have also validated the data type of the props passed to the component.

  - \*\*📄 TaskManagerList.jsx

    > This component will render a number of `TaskManagerItem`s for how many `tasks` we might have.

  - \*\*📄 TaskManagerLogic.jsx

    > This component will render the `InputTaskManager` and `TaskManagerList` components.
    > For now this component will have the `tasks` **array** hardcoded.

---

## \*📂 root/

- \*\*📄 README.md

  > I have professionally documented the project.

- \*\*📄 package.json

  > I have added the `prop-types` dependency.

- ### \*📂 public/

  - \*\*📄 index.html

  > I have added the `Bootstrap` CDN link.

- ### \*📂 src/

  - \*\*📄 index.jsx

    > I have added the `TaskManagerApp` component to the `index.jsx` file.

---

### Thank you for taking the time to review this PR ⭐!

---

If you require additional information or have any questions, please feel free to contact me on Slack as _**Arturo (Arthur) Emanuel Guerra Iturres**_. I'll be happy to assist you. 🎯

---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,10 +37,7 @@
 
 # 📖 Math Magicians <a name="about-project"></a>
```
Comment (ghost) [inline README.md]: Kindly make sure the title corresponds to your actual project (Task Manager)!

File: README.md [hunk 2]
```diff
@@ -49,60 +46,47 @@
...
+    </li>
+    <li>
+      <img src="https://skillicons.dev/icons?i=sass"/>
+      <a href="https://sass-lang.com/">SASS</a>
+    </li>
+    <li>
+      <img src="https://skillicons.dev/icons?i=bootstrap"/>
+      <a href="https://getbootstrap.com/">Bootstrap</a>
+    </li>
+    <li>
```
Comment (ghost) [inline README.md]: Great to see you are going to use Bootstrap! Kindly, at this point of your project I suggest you include it in your Future Features; the same as the logo, until you start applying them.

**Review by HFG43 — CHANGES_REQUESTED** [review]
Hi @ITurres!! Great job so far!! No doubt it is going to be a great project!
I will like to highlight the following:

- [ ] Very detailed pull request!
- [ ] Good distribution of components!
- [ ] Correct gitflow!
- [ ] Linters run ok!
- [ ] Tasks items are rendering ok!

# Required Changes ♻️


### _Please check the comments under the review._

---

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HFG43 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ITurres) [reply README.md]: Silly me! 😮, you are right @HFG43! thank you to point that out Hernan! I'll make the changes right away 🎯

File: README.md (hunk 2 above)
Comment (ITurres) [reply README.md]: ✍🏼 Absolutely right, Bootstrap should be on my **Future Features**. noted! thank you Hernan!

**Review by HFG43 — APPROVED** [review]
@ITurres Great job applying the requested changes!
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Optional suggestions**
**[ ]**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ITurres** [comment]
Hi @HFG43 Hernan! Thank you for reviewing this PR 💯. I wish you a great Thursday and happy coding!👨🏻‍💻

**Pull request merged**